src/cascadia/ut_app/JsonUtilsTests.cpp

```diff
@@ -293,19 +293,39 @@ namespace TerminalAppUnitTests
     // versions.
 
     template<typename TExpected, typename TJson>
-    static void TryBasicType(TExpected&& expected, TJson&& json)
+    static void TryBasicType(TExpected&& expected, TJson&& json, std::optional<std::string> overrideToJsonOutput = std::nullopt)

```

Comment:
shouldn't this be an optional json::value? so you can specify `{1}` or something?

then later you can use `expectedJson[key] = override.value_or(jsonObject);` and simplify the code?

Reply:
Ah. I was trying to do that, but I messed up by making it a `std::optional<TJson>` instead of `std::optional<Json::Value>`. Derp!